lib/csv_row_model/export_collection/csv.rb

```diff
@@ -0,0 +1,32 @@
+module CsvRowModel
+  module ExportCollection
+    class Csv
+      attr_reader :export_model_class, :source_collection
+
+      # @param [Export] export_model export model class
+      def initialize(export_model_class, source_collection)
+        @export_model_class = export_model_class
+        @source_collection = source_collection
+      end
+
+      def header
+        export_model_class.friendly_names
+      end
+
+      def body
+        source_collection.map do |model|
```

Comment:
concerned about memory here--for the architecture.... it maybe to better to architect around a file, rather than a collection. because the file never changes, while the collection will.